game-card.html

```diff
@@ -348,7 +348,7 @@
             }
 
             computeFigureImage() {
-                let rank = GameCard.FIGURES.includes(this.rank) && GameCard.RANKS[this.rank],
+                let rank = GameCard.FIGURES.includes(this.rank.toLowerCase()) && GameCard.RANKS[this.rank.toLowerCase()],
```

Comment:
You probably want to do the same thing in `#computePositionClass` when testing `this.rank === 'a'`.